lib/bonny/server/reconciler.ex

```diff
@@ -185,8 +185,14 @@ defmodule Bonny.Server.Reconciler do
 
     case result do
       {:ok, resources} ->
-        Bonny.Sys.Event.reconciler_fetch_succeeded(measurements, metadata)
-        Enum.each(resources, &reconcile_async(&1, module))
+        Enum.each(resources, fn
+          resource when is_map(resource) ->
+            Bonny.Sys.Event.reconciler_fetch_succeeded(measurements, metadata)
+            reconcile_async(resource, module)
+          {:error, error} ->
```

Comment:
What is the error type here? A struct or an atom. I can add a test for this. Thanks for looking into it!

Reply:
I describe it in #98. Basically the `reconcile/1` was called with `{:error, %HTTPoison.Error{id: nil, reason: :checkout_timeout}}` so that the error in my case was of type `%HTTPoison.Error{}`. But I guess it could be whatever `Base.run()` returns. 

However, the code I wrote does not expect `error` to be of a specific type. So the test would not have to assume anything, don't you think?

I had started writing tests before I gave up. I can commit my WIP if you like. My problem was that `Bonny.Sys.Event` eats up the exceptions. In other words, the test I wrote even succeeds without my fix. I could try something with log capturing...

Let me know what you need. I still have those uncommitted changes.

Reply:
I've committed the tests now - but as I described above, they don't fail.

Reply:
@coryodaniel were you aware of this discussion when merging? I think it is still pending...